src/frontend/src/components/RecordSearch/Record/Question.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import {QuestionData} from './types';
+import {selectAnswer} from '../../../redux/search/actions';
+import store, {AppState} from '../../../redux/store';
+import {connect} from 'react-redux';
+
+interface Props {
+  ambiguous_charge_id: string;
+  question?: QuestionData;
+  loading?: boolean;
+}
+
+class Question extends React.Component<Props> {
+
+  handleRadioChange = (e: React.BaseSyntheticEvent) => {
+    store.dispatch(selectAnswer(e.target.name, e.target.value))
+  };
+
+  render() {
+    if (this.props.question) {
+      const options: { [option: string]: string } = this.props.question.options;
+      return (
+        <div className="w-100 bt bw3 b--light-purple pa3 pb1">
+          <fieldset className="relative mb4">
+            <legend className="fw7 mb2">{this.props.question.question}</legend>
+            <div className="radio">
+            {Object.keys(options).map(
+              (option_str: string) => {
+                const id: string = this.props.ambiguous_charge_id + options[option_str];
+                return (
+                  <div className="dib" key={id}>
+                  <input type="radio" name={this.props.ambiguous_charge_id} id={id} value={options[option_str]} onChange={this.handleRadioChange}/>
+                  <label htmlFor={id}>{option_str}</label>
+                  </div>
+                )
+              })}
+            </div>
+            {(this.props.loading ?
+              <div className="radio-spinner absolute" role="status">
+                <span className="spinner spinner--sm mr1"></span>
+                <span className="f6 fw5">Updating&#8230;</span>
+              </div> :
+                null
+              )
+            }
+          </fieldset>
+        </div>
+      )
+    } else {
+      return (<div></div>);
```

Comment:
What happens if you return nothing?

Reply:
I didn't love this, but `render()` needs to return a react element. The alternative I thought of was to not render a Question component at all if the charge has none, but that requires fetching from the redux store inside Charge, which I didn't love more.

Reply:
oh, cute. `null` is not ok, but `<></>` is, so I'll do that.